### PR TITLE
docs and view page development

### PR DESCRIPTION
**Pull request by bermannoah**
There's a lot going on in this PR:
-- adds feature test gem, actual feature test for docs view page
-- adds and removed a pretty cool little markdown gem (sorries)
-- adds a Spectre CSS stylesheet
-- customizes the above
-- makes the readme a real readme

screen shot 2017-01-03 at 11 37 26 am

@AliSchlereth @MilkMan90 @tmikeschu @nnchambs

### CODE REVIEW

**Comment by tmikeschu** [comment]
Looks awesome, Noah! Thanks for setting this up. Looks good to merge, unless @AliSchlereth or @dijital2016 see something I don't?

File: app/views/docs/index.html.erb [hunk 1]
```diff
@@ -21,11 +21,15 @@
               <td>GET</td>
               <td>https://edualize.herokuapp.com/api/v1/median_expenditures/totals</td>
               <td>retrieve median expenditures for schools</td>
-              <td>this will retrieve all results!</td>
+              <td>this will return a list of expenditures for all 50 state's schools by charter and non-charter schools</td>
```
Comment (bermannoah) [inline app/views/docs/index.html.erb]: @AliSchlereth How's this? Go ahead and merge it if it's good!

**Comment by AliSchlereth** [comment]
Looks great, thanks for the update!

File: app/views/docs/index.html.erb (hunk 1 above)
Comment (AliSchlereth) [inline app/views/docs/index.html.erb]: Thanks!

**Pull request merged**